src/cachemachine/rubinrepoman.py

```diff
@@ -17,6 +17,17 @@
 logger = structlog.get_logger(__name__)
 
 
+def is_real_release(t: str) -> bool:
```

Comment:
This logic looks good but can you add this to the class like the helper _friendly_name?

Reply:
Yeah,  I just figured since it didn't depend on `self` at all it was a better standalone function.  I'll move it.

Reply:
Done.